utils/fixed_decimal/benches/fixed_decimal.rs

```diff
@@ -51,6 +51,8 @@ fn overview_bench(c: &mut Criterion) {
         larger_isize_benches(c);
         to_string_benches(c);
         from_string_benches(c);
+        from_float_ryu_benches(c);
```

Comment:
If it's not too hard, could you run the benches using the String parsing approach you had originally, and compare the performance with the ryu_floating_decimal approach you are using now?